suites/nautilus/rgw/tier-1_rgw_ssl_test-upgrade-4-to-latest.yaml

```diff
@@ -0,0 +1,255 @@
+      module: sanity_rgw.py
+      config:
+        script-name: test_swift_basic_ops.py
+        config-file-name: test_swift_large_download.yaml
+        timeout: 300
+
+  #versioning tests
+  - test:
+      name: Versioning Tests
+      desc: Versioning with copy objects
+      polarion-id: CEPH-9221
+      module: sanity_rgw.py
+      config:
+        script-name: test_versioning_copy_objects.py
+        config-file-name: test_versioning_copy_objects.yaml
+        timeout: 300
+
+  - test:
+      name: Test deletion of object versions
+      desc: test to delete versioning objects
+      polarion-id: CEPH-14262
+      module: sanity_rgw.py
+      config:
+        script-name: test_versioning_with_objects.py
+        config-file-name: test_versioning_objects_delete.yaml
+        timeout: 300
+
+  #resharding tests
+  - test:
+      name: Resharding tests
+      desc: Reshading test - manual
+      polarion-id: CEPH-83571740
+      module: sanity_rgw.py
+      config:
+        script-name: test_dynamic_bucket_resharding.py
+        config-file-name: test_manual_resharding.yaml
+        timeout: 500
+  - test:
+      name: Dynamic Resharding tests
+      desc: Reshading test - dynamic
+      polarion-id: CEPH-83571740
+      module: sanity_rgw.py
+      config:
+        script-name: test_dynamic_bucket_resharding.py
+        config-file-name: test_dynamic_resharding.yaml
+        timeout: 500
+
+  - test:
+      name: Dynamic Resharding tests
+      desc: Reshading test on versioned buckets - dynamic
+      polarion-id: CEPH-83571740
+      module: sanity_rgw.py
+      config:
+        script-name: test_dynamic_bucket_resharding.py
+        config-file-name: test_dynamic_resharding_with_version.yaml
+        timeout: 500
+
+  #lifecycle tests
+  - test:
+      name: Bucket Lifecycle Object_expiration_tests
+      desc: Test object expiration for Prefix and tag based filter and for more than one days
+      polarion-id: CEPH-11179, CEPH-11180
+      module: sanity_rgw.py
+      config:
+        script-name: test_bucket_lifecycle_object_expiration.py
+        config-file-name: test_lc_rule_prefix_and_tag.yaml
+        timeout: 300
+
+  - test:
+      name: Bucket Lifecycle Object_expiration_tests
+      desc: Test object expiration for non current version expiration
+      polarion-id: CEPH-11190
+      module: sanity_rgw.py
+      config:
+        script-name: test_bucket_lifecycle_object_expiration.py
+        config-file-name: test_lc_rule_prefix_non_current_days.yaml
+        timeout: 300
+
+  #upgrade ceph cluster
+  - test:
+      name: Upgrade ceph cluster to 4.x latest
+      polarion-id: CEPH-83574678
+      module: test_ansible_upgrade.py
+      config:
+        ansi_config:
+          ceph_origin: distro
+          ceph_repository: rhcs
+          ceph_rhcs_version: 4
+          osd_scenario: lvm
+          osd_auto_discovery: false
+          copy_admin_key: true
+          radosgw_frontend_ssl_certificate: "/etc/ceph/server.pem"
```

Comment:
Do we need this configuration?

Reply:
yes, it would override with default configuration.

Reply:
Required to maintain status quo